### DISCUSSION TITLE
Branching mechanism seems weird

### DISCUSSION THREAD

**Discussion by Dieterbe** [LunarVim/LunarVim#1881] [General]
Hello,
this may be just me, but all the other projects i'm involved with use master (or main) as the main/rolling development branch.
When they want a more stable branch, they would then use a branch named "stable" or similar and merge master/main into that. (or even use fast forward pulls)

Seems lunarvim does it the opposite way: all main development goes into rolling, and periodic merges into master. This seems counter-intuitive.
As an aspiring contributor i wonder if we can switch it around, or at the very least mention the strategy in the CONTRIBUTING.md file or in other developer docs (i did see a quick note in the install docs but not much detail there)

**Comment by chase** [comment]
If you'd like to contribute to the CONTRIBUTING.md file to update how the current system works, please make a PR.
Personally I don't think it's too terribly confusing, but I can understand that it isn't particularly standard.
